mock/administration.go

```diff
@@ -0,0 +1,37 @@
+package mock
+
+import (
+	"context"
+	"github.com/influxdata/flux"
+	"github.com/influxdata/flux/execute"
+	"github.com/influxdata/flux/memory"
+)
+
+// this struct is incomplete. You can add what you need when you need it.
+type Administration struct {
```

Comment:
This is to be used in the test for `sql.from` and `socket.from` tests.
For those data sources, the test is different from transformations like `kafka.to` and `sql.to`

Reply:
I think this may be a duplicated effort. I added a similar `administration.go` to the `mock` package.

Reply:
Check the master branch to see if this is a similar addition.

Reply:
I have merged the master into my branch and resolved the conflicts. the table_test refactor and the elimination of the MockAllocator will come up soon.